server/util/networking/networking.go

```diff
@@ -54,14 +57,22 @@ func runCommand(ctx context.Context, args ...string) error {
 // namespace prepends the provided command with 'ip netns exec "netNamespace"'
 // so that the provided command is run inside the network namespace.
 func namespace(netNamespace string, args ...string) []string {
-	return append([]string{"ip", "netns", "exec", netNamespace}, args...)
+	return append([]string{"ip", "netns", "exec", netNamespacePrefix + netNamespace}, args...)
+}
+
+// Deletes all of the executor net namespaces. These can be left behind if the
+// executor doesn't exit gracefully. Unfortunately, "ip netns delete" doesn't
+// support patterns, so this runs a short shell for-loop.
+func DeleteNetNamespaces(ctx context.Context) error {
+	return runCommand(ctx, "sh", "-c",
+		"for ns in $(ip netns list | grep ^bb-executor | cut -d \" \" -f 1); do ip netns delete $ns; done")
```

Comment:
what does the `| cut` part do? for me the output is the same with or without it (wonder if we have different versions of `ip`)

```
$ ip netns list | grep ^bb-executor
bb-executor-1
bb-executor-2
$ ip netns list | grep ^bb-executor  | cut -d " " -f 1
bb-executor-1
bb-executor-2
```